src/tools/interop/idt/capture/__init__.py

```diff
@@ -17,14 +17,14 @@
 
 from capture import ecosystem, platform
 
-from .factory import EcosystemCapture, EcosystemController, EcosystemFactory, PlatformFactory, PlatformLogStreamer
+from .controller import EcosystemCapture, EcosystemFactory, Orchestrator, PlatformFactory, PlatformLogStreamer
 from .pcap import PacketCaptureRunner
 
 __all__ = [
     'ecosystem',
     'platform',
     'EcosystemCapture',
-    'EcosystemController',
+    'Orchestrator',
```

Comment:
I would try to keep the entires in alphabetical order

Reply:
This is because of refactor functionality in the IDE. Anyway, resolved this specific instance.
The linters should be taking care of these kind of things, which I've probably not all applied yet. Hopefully autopep8 or some other tool does that and humans do not need to :)?